API/OtrContext.cs

```diff
@@ -240,7 +238,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 			      .HasConstraintName("Tournaments___fkmatchid")
 			      .IsRequired();
 
-			entity.HasIndex(e => new { e.Name, e.Abbreviation }).IsUnique();
+            entity.Property(e => e.SubmitterUserId).IsRequired(false).HasDefaultValue(null);
```

Comment:
Every tournament has to be submitted by someone.

```suggestion
            entity.Property(e => e.SubmitterUserId).IsRequired();
```